product_docs/docs/pgd/4/rel_notes/pgd_4.3.2+p1_rel_notes.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: "Release notes for EDB Postgres Distributed version 4.3.2+p1"
+navTitle: "Version 4.3.2+p1"
+---
+
+EDB Postgres Distributed version 4.3.2+p1 is a patch release of EDB Postgres Distributed 4, which includes bug fixes for issues identified in previous versions.
+
+If you are using any previous release of HARP, we recommend that you upgrade to HARP 2.3.2.
+
+!!! Note
+This version is required for EDB Postgres Advanced Server versions 12.15, 13.11, 14.8 and later.
+!!!
+
+| Component | Version |  Type           |                                                   Description                                                           |
+| --------- | ------- | --------------- | ------------------------------------------------------------------------------------------------------------------------|
+| HARP      | 2.3.2   | Bug fix | Multiple fixes related to network partition in BDR DCS. |
+| HARP      | 2.3.2   | Bug fix | Fix intermittent harp-manager hang issue. |
+| HARP      | 2.3.2   | Bug fix | In Go driver calls, ensure rows.Err() is checked every time after rows.Next() returns false. |
+| HARP      | 2.3.2   | Change | Improve the BDR DCS's List method performance. |
+| HARP      | 2.3.2   | Change | Upgrade 3rd party dependencies to fix Github dependabot alerts. |
```

Comment:
Again, should this internal change be included in customer-facing docs.

Reply:
Removed that line.